src/btree/bt_debug.c

```diff
@@ -1638,6 +1639,12 @@ __debug_ref(WT_DBG *ds, WT_REF *ref)
     if (__wt_ref_addr_copy(session, ref, &addr) && !WT_TIME_AGGREGATE_IS_EMPTY(&addr.ta))
         WT_RET(ds->f(ds, ", %s, %s", __wt_time_aggregate_to_string(&addr.ta, time_string),
           __wt_addr_string(session, addr.addr, addr.size, ds->t1)));
+    if (ref->state == WT_REF_DELETED && ref->page_del != NULL) {
```

Comment:
Just to point out that this code change is not related to RTS. I am fine to change it in this PR. How about changing the PR title to `RTS and dump` logs?

Reply:
Sure thing, done!